lib/SimplePie/SimplePie/Sanitize.php

```diff
@@ -145,6 +146,25 @@ public function pass_file_data($file_class = 'SimplePie_File', $timeout = 10, $u
 		}
 	}
 
+	public function rename_attributes($tags = array())
+	{
+		if ($tags)
+		{
+			if (is_array($tags))
+			{
+				$this->rename_attributes = $tags;
+			}
+			else
+			{
+				$this->rename_attributes = explode(',', $tags);
+			}
+		}
+		else
+		{
+			$this->rename_attributes = false;
+		}
+	}
```

Comment:
You could also change the order like:
```php
if (is_array($tags)) {
    $this->rename_attributes = $tags;
} elseif (is_string($tags)) {
    $this->rename_attributes = explode(',', $tags);
} else {
    $this->rename_attributes = false;
}
```

This will reject everything that is not supported. I don't know if this something we want though. In this case if you have a number, it will not be treated as a string.

Reply:
I don't think it makes sense to reject anything. If it's available as data-whatever you can use it in CSS or JS; if it's not you can't.

Reply:
The first example is not rejecting anything. It's a rewrite without the layers. The second example is just that, an example :)
You're right about not rejecting things at this level.

Reply:
it is copy&paste from `strip_htmltags()` below (lines 168ff).
I would say that it does not make sense to have another structure.